admin-dev/themes/new-theme/js/header.js

```diff
@@ -1,10 +1,12 @@
 import $ from 'jquery';
+import refreshNotifications from './notifications.js';
 
 export default class Header {
   constructor() {
     $(() => {
       this.initQuickAccess();
       this.initMultiStores();
+      this.initNotificationsToggle();
       this.initSearch();
     });
   }
```

Comment:
Maybe we should add some space between methods ?

https://github.com/airbnb/javascript#constructors--tostring